synapse/federation/units.py

```diff
@@ -54,11 +55,13 @@ def get_internal_dict(self) -> JsonDict:
             "destination": self.destination,
         }
 
-    def get_context(self) -> str:
-        return getattr(self, "content", {}).get("org.matrix.opentracing_context", "{}")
+    def get_tracing_context_json(self) -> str:
+        return getattr(self, "content", {}).get(
+            EventContentFields.TRACING_CONTEXT, "{}"
+        )
 
-    def strip_context(self) -> None:
-        getattr(self, "content", {})["org.matrix.opentracing_context"] = "{}"
+    def strip_tracing_context(self) -> None:
+        getattr(self, "content", {})[EventContentFields.TRACING_CONTEXT] = "{}"
```

Comment:
Just giving these better names and using `EventContentFields.TRACING_CONTEXT` so we always grab from the same key.